views/list-job-roles.html

```diff
@@ -22,17 +22,18 @@ <h2>Our Job Roles</h2>
                 <table class="table">
                     <thead>
                         <tr>
-                            <th scope="col">JobRole</th>
+                            <th scope="col">Job Role</th>
+                            <th scope="col">Job Role Specification</th>
                         </tr>
                     </thead>
                     <tbody>
                         {% for jobroles in jobroles %}
                         <tr>
-                            <td>{{ jobroles.jobRole }}</td>
-                            <td><a href='jobroles/{{ jobroles.jobroles }}'>View</a></td>
+                            <td><a href='jobroles/{{ jobroles.jobroles }}'>{{ jobroles.jobRole }}</a></td>
+                            <td><a href="{{ jobroles.jobSpecification }}">View Specification</a></td>
```

Comment:
I assume you don't want to redirect to the job spec, instead redirect to a url that displays the job spec. I don't see a new job spec endpoint anywhere to display the spec, was this added somewhere else?

Reply:
The job spec takes you directly to the sharepoint, these sharepoint links have been stored in the database